try:
    print(1/0)
except:
    print("Ups, coś poszło nie tak.")